packages/xo-web/src/common/select-objects.js

```diff
@@ -602,6 +606,63 @@ export const SelectTag = makeStoreSelect(
   { allowMissingObjects: true, placeholder: _('selectTags') }
 )
 
+export const SelectTag = decorate([
+  provideState({
+    initialState: () => ({
+      editing: false,
+    }),
+    effects: {
+      addTag: (effects, newTag) => (_, { multi, onChange, value }) => {
+        if (newTag === value || (multi && includes(value, newTag))) {
```

Comment:
`value` is an object here. Or an array of objects in `multi` mode.

Reply:
Your high order `SelectTag` isn't wrapped with `makeStoreSelect` so `value` is whatever is passed to the `value` prop.

Reply:
Yes. you're right.